hook/index.rb

```diff
@@ -14,6 +14,7 @@
   me = Docker::Container.me
   containers = Docker::Container.all - [me]
 end
+containers.sort_by! { |c| c.name }
```

Comment:
(I'll write document later though)
You can boot yaichi and confirm new behavior on your machine 😉 

```
$ git clone https://github.com/mtsmfm/yaichi
$ cd yaichi
$ # make sure you don't run process which use 80 port
$ docker-compose up
$ open http://localhost:80
```